loader/src/cmd/CMakeLists.txt

```diff
@@ -29,7 +52,7 @@ set(cmd_script_configured "${CMAKE_CURRENT_BINARY_DIR}/cmd${IGN_DESIGNATION}${PR
 
 # Set the library_location variable to the relative path to the library file
 # within the install directory structure.
-set(library_location "../../../${CMAKE_INSTALL_LIBDIR}/$<TARGET_FILE_NAME:${loader}>")
+set(plugin_exe_location "${CMAKE_INSTALL_PREFIX}/${INSTALL_STANDALONE_EXECUTABLE}/$<TARGET_FILE_NAME:${plugin_executable}>")
```

Comment:
actually, I think we want to keep it as a relative path so that the ruby file is relocatable. The ../../.. is to back out from lib/ruby/ignition (see line 67)